docs/modules/ROOT/nav.adoc

```diff
@@ -27,45 +28,55 @@
 *** xref:user-guide/advanced-client-topics/prevent-resource-leaks.adoc[Prevent resource leaks]
 ** xref:user-guide/camel-integration.adoc[Camel Integration]
 ** xref:user-guide/examples.adoc[Examples]
+
+// security-guide
 * xref:security-guide/index.adoc[Security guide]
-** xref:user-guide/ssl.adoc[SSL, TLS and HTTPS]
-** xref:user-guide/auth.adoc[Authentication and authorization]
-** xref:security-guide/ws-securitypolicy-auth.adoc[WS-SecurityPolicy Authentication]
+** xref:user-guide/ssl-tls-https.adoc[SSL, TLS and HTTPS]
+** xref:user-guide/authentication-authorization.adoc[Authentication and authorization]
+** xref:security-guide/ws-securitypolicy-authentication-authorization.adoc[WS-SecurityPolicy Authentication]
```

Comment:
We would have to add `page-aliases` attribute set to the old name, so that Antora cares for redirects. (I need to test how well `page-aliases` work at all.) Otherwise we'd break external links, which I'd prefer to avoid.